File: 1491-average-salary-excluding-the-minimum-and-maximum-salary/1491-average-salary-excluding-the-minimum-and-maximum-salary.py
class Solution:
    def average(self, salary: List[int]) -> float:
        minEl,maxEl,s=float('inf'),float('-inf'),0
        for num in salary:
            s+=num
            minEl,maxEl=min(num,minEl),max(maxEl,num)
        return (s-minEl-maxEl)/(len(salary)-2)

File: 1491-average-salary-excluding-the-minimum-and-maximum-salary/1491-average-salary-excluding-the-minimum-and-maximum-salary.cpp
class Solution {
public:
    double average(vector<int>& salary) {
        int size=salary.size();
        sort(salary.begin(),salary.end());
        double res=0;
        for(int i=1;i<size-1;i++){
            res+=salary[i];
        }
        return res/(size-2);
    }
};